packages/web-components/fast-components/docs/color.md

```diff
@@ -0,0 +1,143 @@
+
+### Palettes
+Each color recipe operates on a palette. A palette in an array of hexadecimal colors ordered from light to dark. By default, FAST components leverage the `neutralPalette` and the `accentPalette`. 
+
+### Background color
+This is the contextual color in which the recipe is rendering on. In foreground, outline, and divider recipes it is used the target color in which to ensure contrast against. In fill recipes it is sometimes used as the starting location in the appropriate palette to begin resolution.
+
+### Offsets
+Some recipes also leverage offset values, typically for *states* (rest, hovered, active, selected). These offsets are used to retrieve a color at the sum of the offset and some reference index (usually the index of the rest color or the background color in the palette).
+
+## Color Recipes
+### Using Color Recipes
+First, ensure the UI element has a *FASTDesignSystemProvider* ancestor element - this element will *resolve* the recipe for a component within it that declares a dependency on the recipe.
+
+```html
+<fast-design-system-provider use-defaults>
+    <my-element>I use a color recipe!</my-element>
+</fast-design-system-provider>
+```
+
+*For more information on the DesignSystemProvider, see the [*DesignSystemProvider* documentation](fast-foundation/fast-design-system-provider.md)*
+
+
+Next - declare the recipe as a dependent *behavior* of a Web Component's stylesheet. Then use the recipe as a CSS Custom Property in the stylesheet:
+
+```js
+import { css } from "@microsoft/fast-element";
+import { neutralFillRest } from "@microsoft/fast-components";
+
+const styles = css`
+    :host {
+        background: var(--neutral-fill-rest);
+    }
+`.withBehaviors(
+    neutralFillRest
+);
+```
+
+### Layer Recipes
+Layer recipes represent the UI layers and surfaces that individual UI elements are contained within. They are applied to primary regions such as toolbars, sidebars, canvas regions, fly-outs, dialogs, and cards.
+
+| name | Description |
+|------|-------------|
+| `neutralLayerFloating` | Used as the background for floating layers, including context menus and fly-outs. |
+| `neutralLayerCard` | Used as the background color for cards. Pair with `neutralLayerCardContainer` for the container background. |
+| `neutralLayerCardContainer` | Used as the background color for card containers. Pair with `neutralLayerCard` for the card backgrounds. |
+| `neutralLayerL1` | Used as the background color for the primary content layer (L1). |
+| `neutralLayerL1Alt` | Alternate color for L1 surfaces. |
+| `neutralLayerL2` | Used as the background for the top command surface, logically below L1. |
+| `neutralLayerL3` | Used as the background for secondary command surfaces, logically below L2. |
+| `neutralLayerL4` | Used as the background for the lowest command surface or title bar, logically below L3. |
```

Comment:
is there an accentLayer equivalent? I think we should have a short description at the top clarifying that this table belongs to the neutral palette or something so that it's understood why that extra naming is present instead of just being "Floating", "Card", "L1" etc.

Reply:
There are not - I'll just move it under the **Neutral Recipes** heading